internal/service/wafv2/helper.go

```diff
@@ -68,6 +68,7 @@ func wafv2RootStatementSchema(level int) *schema.Schema {
 				"label_match_statement":                 wafv2LabelMatchStatementSchema(),
 				"not_statement":                         wafv2StatementSchema(level - 1),
 				"or_statement":                          wafv2StatementSchema(level - 1),
+				"rate_based_statement":                  wafv2RateBasedStatementSchema(level),
```

Comment:
The usage is the same as web_acl, so I made the level the same.

https://github.com/hashicorp/terraform-provider-aws/blob/b9edd007de21648a07f9e73b0a3ba6ac3f60db95/internal/service/wafv2/web_acl.go#L383